lib/vtt.js

```diff
@@ -818,14 +846,19 @@
       // result in the desired behaviour.
       lh = rects ? Math.max((rects[0] && rects[0].height) || 0, obj.height / rects.length)
                  : 0;
+
     }
     this.left = obj.left;
     this.right = obj.right;
-    this.top = obj.top;
-    this.height = obj.height;
-    this.bottom = obj.bottom;
-    this.width = obj.width;
+    this.top = obj.top || top;
+    this.height = obj.height || height;
+    this.bottom = obj.bottom || (top + (obj.height || height));
+    this.width = obj.width || width;
     this.lineHeight = lh !== undefined ? lh : obj.lineHeight;
+
+    if (isIE8 && !this.lineHeight) {
+      this.lineHeight = 13;
```

Comment:
Yes, I believe that IE8 never has a line height but made the check just in case.